portfolio/src/main/webapp/script.js

```diff
@@ -59,6 +59,35 @@ async function displayComments() {
   }
 }
 
+/**
+ * Displays the login status of the user and a link for logging in or out.
+ */
+async function displayLoginInfo() {
+  const response = await fetch('/login');
+  const loginInfo = await response.json();
+  const loginUrl = loginInfo[0];
+  const userEmail = loginInfo[1];
+
+  const loginContainer = document.getElementById('login-container');
+
+  // Check if the user is logged in.
+  if (userEmail === null) {
+    loginContainer.innerHTML =
+        '<a href="' + loginUrl + '">Click here to log in</a>';
+  } else {
+    loginContainer.innerHTML = 'Logged in as ' + userEmail + '<br /><a href="' +
+        loginUrl + '">Click here to log out</a>';
+  }
+}
+
+/**
+ * Displays comments and login info.
+ */
+async function displayInfo() {
+  await displayComments();
```

Comment:
Why use "await" here?

Reply:
Would removing the await just cause the two functions to run in parallel? And if I do that, do I still need the async keyword?

Reply:
Discussed this with Tony in our 1:1